ksql-engine/src/main/java/io/confluent/ksql/ddl/commands/CommandFactories.java

```diff
@@ -38,60 +38,48 @@
 
   public CommandFactories(
       final KafkaTopicClient topicClient,
-      final QueryTerminator queryTerminator,
+      final MetaStore metaStore,
```

Comment:
We can get rid of this if DropSourceCommand isn't using the metaStore passed to its constructor

Reply:
This was removed by previous PR already.